src/mlpack/methods/ann/ffn.hpp

```diff
@@ -94,15 +94,18 @@ class FFN
    * object, be sure to use std::move to avoid unnecessary copy.
    *
    * @tparam OptimizerType Type of optimizer to use to train the model.
+   * @tparam CallbackTypes Types of Callback Functions
```

Comment:
Just a small silly mistake. Do not forget `periods` after comment statements. :)